lineage/figures/figure6.py

```diff
@@ -88,13 +87,13 @@ def figureMaker5(ax, distribution_df, wasser_df):
 
     i += 1
 
-    sns.violinplot(x="G1 lifetime", y="Distribution similarity", hue="State", palette={"State 1": "b", "State 2": "g"}, split=True, data=distribution_df, ax=ax[i])
+    sns.violinplot(x="G1 lifetime", y="Distribution Similarity", hue="State", palette={"State 1": "b", "State 2": "g"}, split=True, data=distribution_df, ax=ax[i])
 
     i += 1
     # state accuracy
-    sns.regplot(x="Wasserstein distance", y="State Assignment Accuracy", data=wasser_df, label="tHMM", ax=ax[i], lowess=True, marker='+')
-    sns.regplot(x="Wasserstein distance", y="KMeans accuracy", data=wasser_df, ax=ax[i], label="K-means", lowess=True, marker='+')
-    ax[i].set_title("State Assignment Accuracy")
+    sns.regplot(x="Wasserstein distance", y="State Assignment Performance", data=wasser_df, label="tHMM", ax=ax[i], lowess=True, marker='+')
```

Comment:
Accuracy seems like a better term than Performance, which is too vague. To be more specific, this could be "Random Index Accuracy" or some such.

Reply:
Will replace them with "Random Index Accuracy". Thanks.